packages/generators/add-generator.ts

```diff
@@ -1,57 +1,24 @@
 import * as Generator from "yeoman-generator";
-
-import * as glob from "glob-all";
-import * as autoComplete from "inquirer-autocomplete-prompt";
-import * as path from "path";
-
 import npmExists from "@webpack-cli/utils/npm-exists";
 import { getPackageManager } from "@webpack-cli/utils/package-manager";
-import PROP_TYPES from "@webpack-cli/utils/prop-types";
-import { AutoComplete, Confirm, Input, List } from "@webpack-cli/webpack-scaffold";
+import { Input, List } from "@webpack-cli/webpack-scaffold";
+import webpackDevServerSchema from "webpack-dev-server/lib/options.json";
+import * as AutoComplete from "inquirer-autocomplete-prompt";
+import path, { resolve } from "path";
+import glob from "glob-all";
```

Comment:
glob and path on top of import, then the autocomplete. Add a space after that and the cli imports after that

```
const fs ...
const path ....
const autocomplete

const cli..
const cli2..
```